### ISSUE TITLE
Move scss variables into a map (key value pair)

### ISSUE THREAD

**Issue by Jakob-Maudience** [Thejuse/SocialColors#17]
It would be much cleaner to store the scss color variables in a map like so
``` scss
$sc-vars : (
    facebook: #3b5998,
    twitter: #1da1f2,
    ...
);
```

This way, for the class names generation, you only have to make a loop instead of making a separate class name for each brand manually, making your brands.scss file much shorter (14 lines instead of 800)

``` scss
.#{$sc-css-prefix} {
  @each $name, $color in $sc-vars {
    &.#{$sc-css-prefix}-#{name} {
      &.#{$sc-css-prefix}-hover {
        &:hover {
          color: $color;
        }
      }
      &.#{$sc-css-prefix}-color {
        color: $color;
      }
    }
  }
}
```

To get the value directly just use `map-get($sc-vars, 'facebook');`

**Comment by Thejuse** [comment]
Hello Jakob-Maudience,

I‘ve added this now to my ToDoList.

Thanks

**Comment by Thejuse** [comment]
Implemented in the next Version (3.0.0).

Thanks for helping to get socialColors better.
